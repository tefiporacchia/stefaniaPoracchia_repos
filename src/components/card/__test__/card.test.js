import {render, screen, cleanup} from '@testing-library/react';
import React from 'react';
import Card from "../card";

describe('Main page mount', () => {

    it("must display the main page title", () => {
        render(<Card/>);
        expect(screen.getByText("hola")).toBeInTheDocument()
    })

})