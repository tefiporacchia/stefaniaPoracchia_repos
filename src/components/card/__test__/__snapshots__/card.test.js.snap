// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main page mount renders correctly 1`] = `
<div
  className="card-container number-3"
  onClick={[Function]}
>
  <img
    alt="charizard"
    src="https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/other/dream-world/2.svg"
  />
  <div
    className="number"
  >
    <small>
      #0
      3
    </small>
  </div>
  <h3>
    Charizard
  </h3>
</div>
`;
